app/furniture/layouts/marketplace.html.erb

```diff
@@ -1,22 +1,28 @@
 <%- content_for :content do %>
-  <%= turbo_frame_tag(marketplace, data: { turbo_action: :advance }) do %>
-    <%- if policy(marketplace).edit? %>
-      <%= render Marketplace::ManagementComponent.new(marketplace: marketplace) do %>
-        <%= yield  %>
+  <%- if room.hero_image&.upload.present? %>
+    <%= image_tag room.hero_image&.upload&.variant(resize_to_fill: Media::FULL_WIDTH_SHORT), class: "w-full" %>
+  <%- end %>
+  <div class="flex flex-col content-between justify-end mx-auto max-w-7xl sm:px-6 lg:px-8">
+    <div class="grow mx-auto w-full">
```

Comment:
Adds similar chrome styling as the main layout.

Reply:
What is "main"? Do you mean `layouts/application.html.erb`? I suppose it could although if we raise it to that level I wonder what cascading effects it will have on other non-room, non-marketplace pages. 

I'm a little confused on the "stacking" of templates between Rooms and the Marketplace Gizmo. Would need some more time to trace how the hierarchy to understand how to reconcile this styling across both, and/or more screens.

Reply:
Yes, I mean `layouts/application.html.erb`

And yea, the way the layouts "stack" needs a bit of refinement.

Right now, everything (except the `Marketplace::Flyer`) renders within `layouts/application.html.erb`; which probably should have the framing we want for everything.


`layouts/marketplace.html.erb` is intended for wrapping pages within the `Marketplace`, but still inherits from `layouts/application.html.erb`, so it should only include things that are significant on a per-Marketplace basis like the `ManagementComponent` frame, the `turbo_frame_tag` for the `Marketplace`, maybe some `Marketplace` specific navigation (tho I am not sold on that)